packages/checkbox/src/checkbox.tsx

```diff
@@ -150,7 +150,10 @@ export const Checkbox = forwardRef<CheckboxProps, "input">((props, ref) => {
       className={cx("chakra-checkbox", className)}
       {...getRootProps()}
     >
-      <input className="chakra-checkbox__input" {...getInputProps({}, ref)} />
+      <input
+        className="chakra-checkbox__input"
+        {...getInputProps({ tabIndex: props.tabIndex ?? undefined }, ref)}
```

Comment:
I think you can destructure `tabIndex` so it doesn't get duplicated in the `rootProps` and `inputProps`.

Maybe a better place to handle this is in the `useCheckbox` hook itself?